microprofile/fault-tolerance-smallrye/cdi/pom.xml

```diff
@@ -25,15 +25,17 @@
 
     <parent>
         <groupId>org.wildfly</groupId>
-        <artifactId>wildfly-microprofile-fault-tolerance-smallrye</artifactId>
+        <artifactId>wildfly-parent</artifactId>
```

Comment:
Shouldn't you keep the same parent and set the relative path to `../pom.xml`?
`wildfly-microprofile-fault-tolerance-smallrye-cdi` and `wildfly-microprofile-fault-tolerance-smallrye` seems to be closely related to have the same GAV grouping.